src/openrct2/interface/Fonts.cpp

```diff
@@ -26,38 +26,38 @@
 
 #ifndef NO_TTF
 TTFFontSetDescriptor TTFFontMSGothic = { {
-    { "msgothic.ttc", "MS PGothic", 9, 1, 0, 15, nullptr },
-    { "msgothic.ttc", "MS PGothic", 12, 1, 0, 17, nullptr },
-    { "msgothic.ttc", "MS PGothic", 12, 1, 0, 17, nullptr },
-    { "msgothic.ttc", "MS PGothic", 13, 1, 0, 20, nullptr },
+    { "msgothic.ttc", "MS PGothic", 9, 1, 0, 15, 60, nullptr },
```

Comment:
Maybe add a define for these thresholds?

Reply:
Sure! Since this is C++, shall we implement them using `constexpr`? As for their names, what about `HINTING_THRESHOLD_LOW` for 40 and `HINTING_THRESHOLD_MEDIUM` for 60?

Reply:
No idea what a `constexpr` is, but if it's better: sure :P. Names seem fine though.

Reply:
Actually, `constexpr` is probably overkill. I've just defined them as old-skool consts. :)